src/app/user/register/post.controller.js

```diff
@@ -129,9 +129,11 @@ module.exports = (req, res) => {
     })
     .catch((err) => {
       logger.info('Failed registration validations');
-      logger.debug(JSON.stringify(err));
       res.render('app/user/register/index', {
         cookie,
+        fname,
```

Comment:
Passing said variables back to the form so that cookies aren't used to populate instead.